dist/templates/ovnkube-monitor.yaml.j2

```diff
@@ -27,7 +27,7 @@ kind: Service
 metadata:
   labels:
     k8s-app: ovnkube-master
-  name: ovn-kubernetes-master-prometheus-discovery
+  name: ovnkube-master-prometheus-discovery
```

Comment:
this service is still using port `http-metrics` listening in 9409, should this be listening 
> under '/metrics' path at port 9476.

Reply:
same question for the service above, see Line14

Reply:
@aojea didn't follow your question here?

- ovnkube-master Pod, via ovnkube-master daemon, exports ovnkube-master metrics at TCP port 9410 (named as http-metrics)
- ovnkube-node Pod, via ovnkube-node daemon,  exports ovnkube-node metrics at TCP port 9409 and ovn-metrics at TCP Port 9476
- ovnkube-node Pod, via ovs-metrics-exporter container, exports ovs-metrics at TCP Port 9310

Reply:
nah, forget, I was confused with the names and the ports :sweat_smile:

Reply:
no worries